frontend/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Bowtie –</title>
```

Comment:
Is there supposed to be a template variable or something here after the –? I'm no react expert, does something magically append page names to the `<title>` or yeah is there supposed to be something after that?

Reply:
The dialectShortName is being dynamically appended by the following code in `App.js` file.

```
  document.getElementsByTagName("title")[0].textContent +=
    " " + runInfo.dialectShortName;

```